src/trait_manager/model/Trait.js

```diff
@@ -67,6 +67,11 @@ export default Backbone.Model.extend({
 
     if (this.get('changeProp')) {
       target.set(name, value, opts);
+      if (name === 'poster') {
```

Comment:
You're putting some specific condition (the poster attribute from video) in a generic object (Trait) and this is not good from the architectural and logical point of view. At least, you should put that condition in the Video Component